### DISCUSSION TITLE
Homens São Complicados

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#415] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/homens-sao-complicados/

**Comment by brunoaugusto** [comment]
Alguém mais fez os movimentos pra ver se conferem?

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/homens-sao-complicados/#comment_1128851097>):

> Alguém mais fez os movimentos pra ver se conferem?

Yep! Todos conferem!
